### ISSUE TITLE
Create Mathematica builder

### ISSUE THREAD

**Issue by matthiaswd** [gslab-econ/gslab_python#169]
Creates a mathematica builder complete with tests etc.

**Comment by matthiaswd** [comment]
Above commit creates a working builder. Next up:

- create tests
- check that (error) log prints accurately

**Comment by matthiaswd** [comment]
@jmshapir this issue is still at [this point](https://github.com/gslab-econ/gslab_python/issues/169#issuecomment-611999257):

I'd ideally like to add

- a file `test_build_mathematica.py` to the [tests](https://github.com/gslab-econ/gslab_python/tree/fce9a39110b274f4a1ce8c15cbdd901497c8625e/gslab_scons/tests) folder
- have the output from the Mathematica window print to the working directory if the build fails.

**Comment by veli-m-andirin** [comment]
@jmshapir, per [this comment](https://github.com/gslab-econ/gslab_python/issues/169#issuecomment-667682928) we had two goals remaining in this issue. I tackled the second one which is:

> have the output from the Mathematica window print to the working directory if the build fails.

Previously, sconstruct.log in the root and sconscript.log in the target directories did not include any logs about errors. I changed the executable in gslab_scons/misc.py from `MathKernel` to `wolfram` and now error messages are included in logs.

I'll start working on the remaining goal. Could you please give me write-access to this repository?

**Comment by jmshapir** [comment]
@zkashner can you elevate @veli-m-andirin to write permission here? Thanks!

**Comment by zkashner** [comment]
@jmshapir done!

**Comment by veli-m-andirin** [comment]
@zkashner, sorry to bother, but could you check my write permission once again? I get the following error when I try to push changes:

```
remote: Permission to gslab-econ/gslab_python.git denied to veli-m-andirin.
fatal: unable to access 'https://github.com/gslab-econ/gslab_python.git/': The requested URL returned error: 403
```

I can't edit files on GitHub either. I removed my credentials and tried to log in after pushing but that didn't solve the issue.

**Comment by zkashner** [comment]
@veli-m-andirin I looked into it and it says that your invite is still "pending". If you could please go into your email and search for this invitation I believe you need to accept it for you to have write access.
<img width="750" alt="Screen Shot 2021-01-20 at 7 10 30 AM" src="https://user-images.githubusercontent.com/12866049/105186104-96951100-5aee-11eb-9855-f40e60614eac.png">

**Comment by veli-m-andirin** [comment]
Thanks @zkashner! My apologies for having missed this. Since I didn't accept it early enough it expired. Could you please send a new invite?

**Comment by zkashner** [comment]
@veli-m-andirin just resent! Let me know if you get it please.

**Comment by veli-m-andirin** [comment]
@zkashner, accepted, thanks!

**Comment by veli-m-andirin** [comment]
@jmshapir, 

- I've updated the builder. Now it prints a log of error messages to sconscript.log file in the target folder. To do so I changed the executable name from MathKernel to math (changing it to wolfram has the same effect). However, `math` is not used on macOS, so its users would need to change `line 180 in gslab_scons\misc.py` to `'mathematica': 'MathKernel' ` or rename MathKernel as math. (I don't know if the latter is a feasible operation on that OS.)
- I also added a test script.
